### ISSUE TITLE
Weird behaviour and language mix

### ISSUE THREAD

**Issue by derwaldgeist** [TAPevents/tap-i18n#133]
I am using this package for a while, and it worked quite well, thanks.

But now I faced a very weird behaviour in one of my dialogs. Sometimes, this dialog shows up in the correct language, sometimes it reverts to English. And sometimes, I see a strange mix of two languages, in the very same dialog.

I checked the JSON for this dialog, everything seems to be ok, and also the linter does not complain.

What makes it even weirder: If this problem occurs and I force the compiler to rebuild the JSON file (e.g. by removing keys from it), the problem sometime goes away. And if I re-add the very same keys I had removed before, it still works correctly. So everything works now with exact the _same_ JSON file I initially used when the problem occured.

Until next time, when I add another key and things start to become weird again.

Any idea what might cause this?

**Comment by derwaldgeist** [comment]
I have now debugged the situation. It turns out that the keys that won't get translated are just not available in the "resStore". They all belong to the same parent key. How can it happen that *one* parent key is not being loaded, while the rest of the keys in the *same* JSON file have been loaded pretty well?! Strange thing is that it sometimes works, and after a server reload the error is back again.

I cannot see any problem with the JSON keys themselves (double-checked them). And also keys that come right after the problematic parent key are loaded, so I don't think it's a JSON parse error.

**Comment by derwaldgeist** [comment]
This problem is completely indeterministic. I tried to track it down to remove keys one by one. Still, sometimes the problem occured, sometimes not. I now separated the JSON in two separate files which together include exact the same keys as before - and the problem disappeared. Maybe a race condition white building the files? But still would not explain why exactly one key was affected.

**Comment by JeremySaks** [comment]
+1.

I've been seeing similar behavior where certain keys occasionally do not get added to the store. There are no syntax problems with the JSON.

**Comment by derwaldgeist** [comment]
It's now getting even weirder. Because I suspected that the problem might have something to do with HTML tags I included in some of the translations, I removed all of them. But they still show up in the app! They are definitely gone in all of the files, but I can still see them. I first thought this might be related to some browser caching, but this also happens on browsers on which I never had viewed my app before. Maybe it's some weird caching behaviour of Meteor's build process? I remember Slava writing in one thread that tap:i18n is still using an "old" build API. I don't know, but the phenomenon is really strange and makes me grow grey hairs now...
